test/project/git/LockfileLibrary.test.ts

```diff
@@ -7,14 +7,19 @@ import {
 
 describe("exploratory tests for proper-locking", () => {
```

Comment:
This test seems to just test lockfile, not our code.

Reply:
Indeed it does.  I don't believe it has any value - but I wasn't sure if it was put there to be a "canary in the mine" as it turned out to be in the failure from #225.  Should I just remove this?